### ISSUE TITLE
Add trace to 'Deprecated' message

### ISSUE THREAD

**Issue by sunny-lan** [NodeRedis/node_redis#1369]
Feature request:

When it shows a message like:

```
node_redis: Deprecated: The HGET command contains a "null" argument.
This is converted to a "null" string now and will return an error from v.3.0 on.
Please handle this in your code to make sure everything works as you intended it to.
```

It is hard to find where the issue is coming from. It would be useful to have a stack trace generated every time this happens. Especially if the dev has no choice over what redis version is used.

**Comment by stockholmux** [comment]
This a reasonable request. Cloud be integrated with [RedisClient.prototype.warn](https://github.com/NodeRedis/node_redis/blob/04a09aa2ffcbae9f380b61a4afe15badfe5be5ef/index.js#L335) as a one-liner by using `console.trace`. The only trick would be it would only be listed in the terminal, not from someone listening for errors, but that seems like it's ok.

**Comment by sunny-lan** [comment]
Wouldn't the `console.trace` have to be put outside of `nextTick`?

**Comment by stockholmux** [comment]
Good point. I think it would still tell you the location but it wouldn't be at the top level of the trace. Need to think on it, but if you have a suggestion, let's hear it.

**Comment by ad34** [comment]
+1 on this. sometimes it s tricky to find the code leading to this warning.

**Comment by 1blockologist** [comment]
Yes, or a way to disable these deprecation warnings on production, a real crash will produce a stack trace anyway if an upgraded version ever actually comes out!

These messages fill up my server logs!
